src/kmedoids_ucb.cpp

```diff
@@ -242,7 +258,7 @@ void KMedoids::setLogFilename(std::string new_lname) {
  * @param input_data Input data to find the medoids of
  * @param loss The loss function used during medoid computation
  */
-void KMedoids::fit(arma::mat input_data, std::string loss) {
+void KMedoids::fit(arma::mat input_data, std::string loss) { 
```

Comment:
remove extraneous whitespace at EOL (for future, can configure a linter automatically to do this)